docs/content/en/docs/pipeline/operation.md

```diff
@@ -101,12 +100,10 @@ el-test-predefined-task-listener-7d5774d7c-82whm   1/1     Running   0
 
 ## Exposing Services and Webhook Settings
 
-Within the Kubernetes cluster, both `el-test-custom-task-listener` and `el-test-predefined-task-listener` are configured as ClusterIP services, 
-meaning they are only accessible internally. 
-To enable external systems, like webhooks, to access these services, we need to expose them outside the cluster. 
+Within the Kubernetes cluster, both `el-test-custom-task-listener` and `el-test-predefined-task-listener` are configured as ClusterIP services, meaning they are only accessible internally.
```

Comment:
Why merge the original content? For code reading, a line that is too long is difficult to read.